manifests/params.pp

```diff
@@ -0,0 +1,15 @@
+# ex: syntax=puppet si ts=4 sw=4 et
+
+class bind::params (
```

Comment:
Yes, I see why this is an issue. Not sure why we've done this, I will revise this request.